utils.h

```diff
@@ -16,6 +16,7 @@
 #define UTILS_H_
 
 #include <string>
+#include <cstdint>
```

Comment:
It might be best to keep the includes in alphabetical order.

Reply:
Done.